bode/bode/resources/tasks/schemas.py

```diff
@@ -1,10 +1,24 @@
+from enum import Enum
+
 from marshmallow import EXCLUDE, fields, validate
 
 from bode.models.task import TaskStatus
 from bode.resources.base_schema import BaseSchema
 from bode.resources.tags.schemas import TagInputSchema, TagSchema
 
 
+class DirectedRelationType(Enum):
+    Blocks = "blocks"
+    IsBlockedBy = "is_blocked_by"
+    Subtask = "subtask"
+    Supertask = "supertask"
+    Interchangable = "interchangable"
+
+    @classmethod
+    def list(cls):
+        return [c.value for c in cls]
+
+
```

Comment:
This class was copied from task_relations.schema.py because of cycle import, am I right?
I suggest do small refactoring (e.g. create file with this Enum Class and import to schemas from this file).